ball_tracker_rgb:
  ros__parameters:
    image_topic: "/camera/image_raw"
    camera_info_topic: "/camera/camera_info"
    # Tennis ball HSV (tune later to your lighting/ball)
    hsv_lower: [25, 120, 120]
    hsv_upper: [45, 255, 255]
    min_area_px: 150
    det_ball_topic: "/detections/ball"
    debug_image_topic: "/camera/image_debug"
    mask_image_topic: "/camera/image_mask"

people_detector:
  ros__parameters:
    image_topic: "/camera/image_raw"
    camera_info_topic: "/camera/camera_info"
    ann_topic: /camera/annotations
    people_debug_topic: /camera/people_debug
    det_topic: "/detections/person"
    model_path: "/ws/mecanumbot_camera/models/yolov8n.onnx"  # <â€” inside the container
    input_size: [640, 640]
    conf_threshold: 0.25
    iou_threshold: 0.45
    infer_every_n: 2
    person_class_ids: [0]

overlay_fused:
  ros__parameters:
    image_topic: /camera/image_raw
    person_det_topic: /detections/person
    ball_det_topic: /detections/ball
    out_topic: /camera/fused_debug